docs/user/aws/route53.md

```diff
@@ -0,0 +1,107 @@
+# Route53
+
+Amazon’s Route53 service is used by the OpenShift installer to configure cluster DNS resolution and provide name lookup 
+for the cluster to the outside world. To use OpenShift, you must have created a public hosted zone in Amazon Route53 in 
+the same account as your OpenShift cluster. You must also ensure the zone is “authoritative” for the domain. There are 
+two ways to do this outlined below: root domain and subdomain. A root domain is “openshiftcorp.com”. A subdomain is of 
+the form “clusters.openshiftcorp.com”.
+
+The below sections identify how to ensure your hosted zone is authoritative for a domain.
+
+## Step 1: Acquire/Identify Domain
+
+You may skip this step if using an existing domain and registrar. You will move the authoritative DNS to Route53 or 
+submit a delegation request for a subdomain in a later step.
+
+Route53 can also purchase domains for you and act as a registrar. If you allow Route53 to purchase a new domain for you,
+you can skip the remainder of these steps (the domain is created and the hosted zone is created correctly for you)!
+
+### Example: Purchasing a new domain
+
+![Domain purchased in Route53 registrar](images/route53_registrar.png)
+
+Later:
+
+![Automatic hosted zone in Route53](images/route53_hosted_zone.png)
+
+## Step 2: Create Public Hosted Zone
+
+Whether using a root domain or a subdomain, you must create a public, hosted zone.
+
+[AWS: Creating a Public Hosted Zone][create-hosted-zone]
+
+To use the root domain, you’d create the hosted zone with the value “openshiftcorp.com”. To use a subdomain, you’d 
+create a hosted zone with the value “clusters.openshiftcorp.com”. (Use appropriate domain values for your situation.)
+Example: Root Domain
+
+![Create hosted zone in Route53](images/route53_create_hosted_zone.png)
+
+
+## Step 3: Get Public Nameservers of Hosted Zone
+
+For either a root domain “openshiftcorp.com” or a subdomain “clusters.openshiftcorp.com”, you must extract the new 
+authoritative nameservers from the hosted zone records.
+
+[AWS: Getting the Name Servers for a Public Hosted Zone][get-hosted-zone-info]
+
+### Example: Root Domain
+
+![Get hosted zone info from Route53](images/route53_hosted_zone_info.png)
+
+## Step 4a: Root Domain - Update Registrar
+
+Each registrar requires a slightly different procedure. Using the four (4) nameserver values from the previous step, 
```

Comment:
Do we really need "four (4)"?  I'd expect folks to be able to read "four", and this isn't a legal document with lots of historical precedent boxes to tick off ;).

Reply:
Fixed. Thanks!